Dockerfile

```diff
@@ -1,23 +1,22 @@
 # https://hub.docker.com/_/alpine
-FROM alpine:3.12
+# Create base image
+FROM alpine:3.12 as base
 
 
 # Install dependencies
-RUN apk add --update --no-cache \
-            ca-certificates \
-            libpcap \
-            libgcc libstdc++ \
-            libressl3.1-libcrypto libressl3.1-libssl \
- && update-ca-certificates \
```

Comment:
I don't think this is a good idea to remove fresh certificates from the final image.

Reply:
I have the impression that `update-ca-certificates` does nothing in alpine 3.12